### PR TITLE
teams: 1.4.00.26453 -> 1.5.00.10453

### PR DESCRIPTION
**Pull request by srhb**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/teams/default.nix [hunk 1]
```diff
@@ -37,7 +37,7 @@ let
 
     src = fetchurl {
       url = "https://packages.microsoft.com/repos/ms-teams/pool/main/t/teams/teams_${version}_amd64.deb";
-      sha256 = "0ndqk893l17m42hf5fiiv6mka0v7v8r54kblvb67jsxajdvva5gf";
+      sha256 = "sha256-fLVw2axSMetuaoRzjg+x4DRYY8WP5TQbL7LbfF6LFfA=";
```
Comment (liff) [inline pkgs/applications/networking/instant-messengers/teams/default.nix:40]:
```suggestion
      hash = "sha256-fLVw2axSMetuaoRzjg+x4DRYY8WP5TQbL7LbfF6LFfA=";
```

**Review by liff — APPROVED** [review]
I don’t currently use Teams so have not tested properly but running the binary works.

**Review by liff — APPROVED** [review]

**Comment by liff** [comment]
@srhb I realized that the hash of the macOS package should have been updated as well; fixed in #171295.

**Comment by srhb** [comment]
Sorry, completely missed there were two hashes. I'll merge tomorrow. Thanks for the heads-up. :)

**Comment by drupol** [comment]
Hey hi!

I'm not able to do `nixos-rebuild` with the latest `nixos-unstable` commit 2a3aac479caeba0a65b2ad755fe5f284f1fde74d.

Its blocking when installing Teams, do you have any clue what it could be ?

```
$ sudo nixos-rebuild switch --upgrade-all --flake github:drupol/nixos-x260                                                                                                                            
building the system configuration...
error: builder for '/nix/store/p928mj0z0qjm7im0i6gafckiylxpnlvp-teams-1.5.00.10453.drv' failed with exit code 1;
       last 10 log lines:
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/native-utils/build/Release/native-utils.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/slimcore/bin/trouter-client.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/slimcore/bin/slimcore.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/slimcore/bin/sharing-indicator.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/slimcore/bin/platform.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/@microsoft/fasttext-languagedetector/build/Release/fastText-languagedetector.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/media-hid/build/Release/media-hid.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/@msteams/node-locale-info-provider/build/Release/node-locale-info-provider.node
       > Adding runtime dependencies to RPATH of Node module /nix/store/4g2smhfmjq3grq8ac0zwgl8jip9p4rb3-teams-1.5.00.10453/opt/teams/resources/app.asar.unpacked/node_modules/@msteams/electron-modules-package-utils/build/Release/package-utils.node
       > consumeEntire(): ERROR: Input null bytes, won't process
       For full logs, run 'nix log /nix/store/p928mj0z0qjm7im0i6gafckiylxpnlvp-teams-1.5.00.10453.drv'.
error: 1 dependencies of derivation '/nix/store/g09zxhng4v08bahjhlq3iaqhrav63yw5-man-paths.drv' failed to build
error: 1 dependencies of derivation '/nix/store/3q9pp8i9bl8mkkyn2wmw1wkhhqihiqca-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/mk1c082rmrcfzvapwls3l3qc2gyh7inn-teams-1.5.00.10453_fish-completions.drv' failed to build
error: 1 dependencies of derivation '/nix/store/dvf6y7s126yg4z6a0p4g61kagj4w6hqk-nixos-system-x260-22.05.20220509.2a3aac4.drv' failed to build
$                       
```

**Comment by srhb** [comment]
No, but I'm fairly certain it's not related to this update. I'll try and remember to bisect later, but I think it might be crossing a staging merge boundary, so probably some global update.

**Comment by srhb** [comment]
Looks similar to what #172315 fixes

**Comment by ncfavier** [comment]
https://github.com/NixOS/nixpkgs/blob/039f1fe1f0ec4ce12a82ed88fc251f2d2fd32849/pkgs/applications/networking/instant-messengers/teams/default.nix#L123-L126

This relies on the wrapper being a shell script, which was changed by https://github.com/NixOS/nixpkgs/pull/164163.

**Comment by ncfavier** [comment]
Fixed in https://github.com/NixOS/nixpkgs/pull/172319

**Pull request merged**